scene/resources/texture.h

```diff
@@ -65,6 +65,10 @@ class Texture2D : public Texture {
 	GDVIRTUAL4C(_draw, RID, Point2, Color, bool)
 	GDVIRTUAL5C(_draw_rect, RID, Rect2, bool, Color, bool)
 	GDVIRTUAL6C(_draw_rect_region, RID, Rect2, Rect2, Color, bool, bool)
+	GDVIRTUAL4C(_get_rect_region, Rect2, Rect2, Rect2, Rect2)
```

Comment:
You can't bind it like this if you want the last two arguments to be return values, you'd need to return a `Dictionary` with the data in it

Reply:
alright, i think i did this correctly in the new commit